core/user_settings.py

```diff
@@ -59,6 +44,40 @@ def get_list_from_csv(
 
     return mapping
 
+def write_csv_defaults(path: Path, headers: tuple[str, str], default: dict[str, str]=None) -> None:
+    if not path.is_file() and default is not None:
+        with open(path, "w", encoding="utf-8", newline="") as file:
```

Comment:
Prefer earlier return for simplicity

```suggestion
    if default is None or path.is_file():
        return
```